.travis.yml

```diff
@@ -3,6 +3,14 @@ language: julia
 os:
   - linux
   - osx
+  - windows
+arch:
+  - x64
+  - x86
+matrix:
+  allow_failures:
+    - julia: nightly
+    - arch: x86
```

Comment:
I'm allowing failure on 32-bit Linux because #18 doesn't solve all the issues, but it does solve the problem it's meant to fix.